instruqt-tracks/sentinel-for-terraform-v4/07-restrict-images-v2/assignment.md

```diff
@@ -0,0 +1,107 @@
+---
+slug: restrict-images-v2
+id: rkzydik0ncoe
```

Comment:
Let's clear the `id:` fields out of `assignment.md` while we're moving to v3 of the track config